WalletWasabi.Backend/Controllers/BlockchainController.cs

```diff
@@ -419,5 +419,33 @@ private async Task<EstimateSmartFeeResponse> GetEstimateSmartFeeAsync(int target
 
 			return feeResponse;
 		}
+
+		[HttpGet("status")]
```

Comment:
If you want to add more info then it is okay, otherwise the name should be something more specific to filters.

Reply:
Yes, I would like to build it in a way that later we can add more status indicators.